### ISSUE TITLE
(One#26772283) [Toggle Minimap]: Toggle Minimap screen is not accessible through Keyboard

### ISSUE THREAD

**Issue by ek68794998** [Azure/LogicAppsUX#4294]
### **User Experience:** 

Users who rely on keyboard for navigation will not be able to access the controls and will not be able to perform the required action on the controls if any interactive controls present on page are not keyboard accessible. As a result, they will miss the functionality.

### **Pre-requisites:**

1. First we need to create a flow to test this feature, a person can create their own flow. 

### **Repro Steps:** 

1. Open URL: [https://make.powerautomate.com/](<https://make.powerautomate.com/>) and sign in with Valid credentials. 
2. Flow Homepage will appear on the screen, navigate to 'My Flows' from the left pane and select it. 
3. My Flows Page will appear on the screen which consist of a list of already created flows. 
4. Navigate to one of the flows on the list and click the icon for "Edit." You are now in the flow designer. 
5. Verify whether Toggle Minimap screen is accessible through keyboard after invoking it. 

### **Actual Result:** 

When Toggle Minimap is invoked a screen is appeared beside that to change the place of workflow design, is not accessible through Keyboard.

**Observation:** Toggle Minimap screen is accessible through mouse, user can change the place of workflow design with mouse but not with keyboard.

### **Expected Result:** 

When Toggle Minimap is invoked a screen is appeared beside that to change the place of workflow design. it should be accessible through Keyboard.

### **ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:** 

\*\*

[https://www.w3.org/WAI/ARIA/apg/patterns/tabs/](<https://www.w3.org/WAI/ARIA/apg/patterns/tabs/>)

### **UIA/DOM details available:** 

\*\*

N/A

\*\*

### **MAS Reference:** 

[MAS 2.1.1 -- Keyboard](<https://aka.ms/MAS2.1.1>)

### **Reference Links:** 

- [Accessibility Insights!](<https://accessibilityinsights.io/>) - Identify accessibility bugs before check-in and make bug fixing faster and easier. 

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

![MAS2 1 1 Toggle Minimap screen is not accessible through Key board](<https://github.com/Azure/LogicAppsUX/assets/1350074/eaa1e954-26b7-482d-9c85-c0af073d9689>)

### Browser 

- **OS Version:** Windows 11 Version 23H2 (OS Build 22631.3007) 
- **Browser:** Edge Version 120.0.2210.144 (Official build) (64-bit) 
- **\#URL:**[https://make.powerautomate.com/](<https://make.powerautomate.com/>) 

### Additional context 

[One#26772283](<https://dev.azure.com/msazure/One/_workitems/edit/26772283>)

**Comment by hartra344** [comment]
This control is not built/intended to be keyboard accessible nor is it in anyway required for use of the designer navigation. It is purely secondary. More context was given in the group chat.
